.github/workflows/main.yml

```diff
@@ -332,6 +316,40 @@ jobs:
       env:
         RUST_BACKTRACE: 1
 
+  # Build and test the C interface
+  test_cmake:
+    name: Test C Interface
+    runs-on: ${{ matrix.os }}
+
+    strategy:
+      matrix:
+        include:
+          - os: ubuntu-latest
+          - os: windows-latest
+
+    steps:
+    - uses: actions/checkout@v3
+      with:
+        submodules: true
+    - uses: ./.github/actions/install-rust
+
+    - run: cargo fetch --locked
+    - run: cargo fetch --locked --manifest-path crates/test-programs/wasi-tests/Cargo.toml
```

Comment:
It's ok to skip these steps since that's only required for one builder to do it.